File: 241006/연속되는 수 2/continuous-number2.py
M = 0

N = int(input())
arr = [int(input()) for i in range(N)]
for i in range(N):
    if i == 0 or arr[i] != arr[i-1]:
        cnt = 1
        for j in range(i+1, N):
            if arr[i] == arr[j]:
                cnt += 1
            else:
                break
        M = max(M, cnt)

print(M)

File: 250106/연속되는 수 2/continuous-number2.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int N, arr[1000];
    cin >> N;
    for (int i=0; i<N; i++) {
        cin >> arr[i];
    }
    int cnt = 0;
    int max = 0;
    for (int i=0; i<N; i++) {
        if (i == 0 || arr[i] != arr[i-1]) {
            if (max < cnt) max = cnt;
            cnt = 1;
        } else {
            cnt++;
        }
    }
    cout << max;
    return 0;
}